src/types/mod.rs

```diff
@@ -16,6 +16,8 @@ pub use self::datetime::{
     PyTzInfo, PyTzInfoAccess,
 };
 pub use self::dict::{IntoPyDict, PyDict};
+#[cfg(not(PyPy))]
+pub use self::dict::{PyDictItem, PyDictKeys, PyDictValues};
```

Comment:
I think you probably wanted the struct `PyDictItems` rather than the `PyDictItem` trait:

```suggestion
pub use self::dict::{PyDictItems, PyDictKeys, PyDictValues};
```

Reply:
Yup thank you so much!